tools/functional-tester/etcd-runner/main.go

```diff
@@ -43,6 +45,7 @@ func main() {
 	endpointStr := flag.String("endpoints", "localhost:2379", "endpoints of etcd cluster")
```

Comment:
@xiang90 sure will do it and raise it in another PR as you suggested.
Thank you!